src/sql/workbench/common/editor/query/queryResultsInput.ts

```diff
@@ -12,15 +12,15 @@ import { QueryPlanState } from 'sql/workbench/common/editor/query/queryPlanState
 import { GridPanelState } from 'sql/workbench/common/editor/query/gridTableState';
 import { QueryModelViewState } from 'sql/workbench/common/editor/query/modelViewState';
 import { URI } from 'vs/base/common/uri';
-import { QueryPlan2State } from 'sql/workbench/common/editor/query/queryPlan2State';
+import { ExecutionPlanState } from 'sql/workbench/common/editor/query/executionPlanState';
 
 export class ResultsViewState {
 	public readonly gridPanelState: GridPanelState = new GridPanelState();
 	public readonly chartState: ChartState = new ChartState();
 	public readonly queryPlanState: QueryPlanState = new QueryPlanState();
```

Comment:
executionPlanState?

Reply:
That belongs to the old query plan feature. That is not completely removed as of now.